docs/testrunner-toolkit/configuration/testcafe.md

```diff
@@ -132,26 +132,135 @@ __Example__:
 
-<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small><a href="/testrunner-toolkit/running-tests#test-on-sauce-labs">ℹ</a></p>
+__Type__: *boolean*
 
-__Description__: Set browser window screen resolution.
+__Example__:
+```yaml
+  disableScreenshots: true
+```
+
+### `tsConfigPath`
+__Description__: The absolute or relative path to the TypeScript configuration file. Relative paths are resolved against the current directory (the directory from which you run TestCafe).
 
 __Type__: *string*
 
 __Example__:
 ```yaml
-    screenResolution: "1920x1080"
+  tsConfigPath: /path/to/file
+```
+
+### `clientScripts`
+__Description__: Injects scripts into all pages visited during the test. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/test-api/fixture/clientscripts.html).
+
+__Type__: *object*
+
+__Example__:
+```yaml
+  clientScripts: ["/path/to/file1", "/path/to/file2"]
+```
+
+### `skipJsErrors`
+__Description__: Ignores JavaScript errors on a webpage. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#skipjserrors).
+
+__Type__: *boolean*
+
+__Example__:
+```yaml
+  skipJsErrors: true
+```
+
+### `quarantineMode`
+__Description__: Enables the quarantine mode for tests that fail. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#quarantinemode).
+
+__Type__: *boolean*
+
+__Example__:
+```yaml
+  quarantineMode: true
+```
+
+### `skipUncaughtErrors`
+__Description__: Ignores uncaught errors and unhandled promise rejections in test code. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#skipUncaughtErrors).
+
+__Type__: *boolean*
+
+__Example__:
+```yaml
+  skipUncaughtErrors: true
+```
+
+### `selectorTimeout`
+__Description__: Specifies the time (in milliseconds) within which selectors attempt to return a node. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#selectorTimeout`).
+
+__Type__: *int*
+
+__Example__:
+```yaml
+  selectorTimeout: 1000
 ```
+
+### `assertionTimeout`
+__Description__: Specifies the time (in milliseconds) TestCafe attempts to successfully execute an assertion if a selector property or a client function was passed as an actual value. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#assertionTimeout).
+
+__Type__: *int*
+
+__Example__:
+```yaml
+  assertionTimeout: 1000
+```
+
+### `pageLoadTimeout`
+__Description__: Specifies the time (in milliseconds) passed after the DOMContentLoaded event, within which TestCafe waits for the window.load event to fire. [See link in Testcafe](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#pageLoadTimeout).
+
+__Type__: *int*
+
+__Example__:
+```yaml
+  pageLoadTimeout: 1000
+```
+
+### `stopOnFirstFail`
```

Comment:
Does this setting conflict with `quaratineMode` if they are both set to true, or does this one only take effect if the test is still determined as failing after quaratineMode is executed?

Reply:
I'm not sure actually 🤔 These fields are all from testcafe config. I'd assume the customer should have known what they want to do when setting these parameters.